### PR TITLE
DOCSP-31380 Migrate kafka docs (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Migrated Kafka docs from old repo to new repo

STAGING: https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-31380/installation/install-kafka/install-kafka-cluster/

JIRA: https://jira.mongodb.org/browse/DOCSP-31380?filter=-1

BUILD: https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64b6bdce87b3b3604672df62

### CODE REVIEW

File: source/installation/install-kafka/kafka-intro.txt [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+--------
+
+Relational Migrator works with Kafka as a Kafka Connect sink connector. A 
+`Debezium connector <https://debezium.io/documentation/reference/stable/connectors/index.html>`__ 
+extracts data from a SQL database and is an event source. Relational Migrator 
+interprets and transforms events from the Debezium source and sends them to a 
+MongoDB cluster which acts as a data sink.
+
+Relational Migrator is not packaged with Kafka Connect. To use Relational Migrator 
+with Kafka you have to install Relational Manager and integrate it with your
```
Comment (sarah-olson-mongodb) [inline source/installation/install-kafka/kafka-intro.txt:36]: ([imperative mood](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/))
In this case it's probably fine to just say something like:
"To use Relational Migrator with Kafka, install Relational ..."

File: source/installation/install-kafka/kafka-intro.txt [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  have your own approach to installing and configuring Kafka.
+
+- :ref:`Install with Docker <install-docker>`
+
+  Install with Docker if you are using a containerized cluster. 
+
+- :ref:`Install with Kubernetes <install-kubernetes>`
+
+  Install with Kubernetes if you need a production-scale option and have experience 
+  with Kubernetes.
```
Comment (sarah-olson-mongodb) [inline source/installation/install-kafka/kafka-intro.txt:63]: (suggestion)
Let's just say "...if you need a production-scale option."

File: source/installation/install-kafka/install-kubernetes.txt [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator with Kafka using Kubernetes. Relational Migrator 
+with Kafka is intended for large migration jobs that typically require long run 
+times. This installation can be used in either POC, testing, or production 
+environments. 
+
+About This Task
```
Comment (sarah-olson-mongodb) [inline source/installation/install-kafka/install-kubernetes.txt:20]:
```suggestion
About this Task
```

File: source/installation/install-kafka/install-kubernetes.txt [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  Kafka with Kubernetes installation process by providing various configuration 
+  options not provided by other installation methods. 
+
+- Use minikube to test your installation locally. For more information, see the 
+  `minikube documentation <https://minikube.sigs.k8s.io/docs/start/>`__.
+
+- If you are using Kubernetes in a production environment, see the 
+  `Kubernetes production considerations <https://kubernetes.io/docs/setup/production-environment/>`__.
+
+Before You Begin
```
Comment (sarah-olson-mongodb) [inline source/installation/install-kafka/install-kubernetes.txt:33]:
```suggestion
Before you Begin
```

File: source/installation/install-kafka/install-docker.txt [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator with Kafka using Docker. Relational Migrator 
+with Kafka is intended for large migration jobs that normally require long run 
+times. The Docker installation method is ideal if you are using a containerized
```
Comment (sarah-olson-mongodb) [inline source/installation/install-kafka/install-docker.txt:17]:
```suggestion
times. The Docker installation method is ideal if you want a containerized 
```

**Review by sarah-olson-mongodb — APPROVED** [review]
Hey @jocelyn-mendez1. Very nice work. Just a couple nits and this LGTM!

File: source/installation/install-kafka/kafka-intro.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/installation/install-kafka/kafka-intro.txt]: Done.

File: source/installation/install-kafka/kafka-intro.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/installation/install-kafka/kafka-intro.txt]: Done.

File: source/installation/install-kafka/install-kubernetes.txt (hunk 3 above)
Comment (jocelyn-mendez1) [reply source/installation/install-kafka/install-kubernetes.txt]: Done.

File: source/installation/install-kafka/install-kubernetes.txt (hunk 4 above)
Comment (jocelyn-mendez1) [reply source/installation/install-kafka/install-kubernetes.txt]: Done.

File: source/installation/install-kafka/install-docker.txt (hunk 5 above)
Comment (jocelyn-mendez1) [reply source/installation/install-kafka/install-docker.txt]: Done.

**Pull request closed without merging**